### PR TITLE
Change handle_packet_in method behavior

### PR DESCRIPTION
**Pull request by gleybersonandrade**
This PR changes handle_packet_in method to check if the ports inside actions list aren't disable from 'config' field (i.e.: OFPPC_NO_FWD flag)  when sending a packet_out message.

Fix kytos/of_core#79

### CODE REVIEW

**Comment by gleybersonandrade** [comment]
> Couldn't the check be in `_create_packet_out`?

It can be, it's really better. I'll change this.

Thanks @ajoaoff

**Review by josemauro — APPROVED** [review]

**Comment by italovalcy** [comment]
Hi guys,

The patch seems to be very good for me, except for the lack of logs, exceptions, or any other information that could help the kytos user to know that the packet_out is not being sent due to the port disable. Don't you guys think it should register a log event saying the packet_out was not sent and why? Especially here: https://github.com/kytos/of_l2ls/pull/19/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR133-R135

File: main.py [hunk 1]
```diff
@@ -103,16 +105,36 @@ def _create_flow_mod(version, packet, port):
...
+    def _create_packet_out(version, packet, ports, switch):
         """Create a PacketOut message with the appropriate version and data."""
         if version == '0x01':
-            packet_out = PacketOut10()
             port = ports[0] if ports else Port10.OFPP_FLOOD
+
+            try:
+                iface = switch.get_interface_by_port_no(port.value)
+            except AttributeError:
+                iface = switch.get_interface_by_port_no(port)
```
Comment (beraldoleal) [inline main.py:116]: My only concern is here, you are querying twice, in the worst-case scenario. IIUC, AttributeError is not raised by `get_interface_by_port_no`, so my suggestion is something like this:
```python
try:
   port_number = port.value
except AttributeError:
   port_number = port
iface = switch.get_interface_by_port_no(port_number)
```

Or we should handle this properly inside `get_interface_by_port_no`.

File: main.py [hunk 2]
```diff
@@ -103,16 +105,36 @@ def _create_flow_mod(version, packet, port):
...
             packet_out.actions.append(Output10(port=port))
 
         else:
-            packet_out = PacketOut13()
             port = ports[0] if ports else Port13.OFPP_FLOOD
+
+            try:
+                iface = switch.get_interface_by_port_no(port.value)
+            except AttributeError:
+                iface = switch.get_interface_by_port_no(port)
```
Comment (beraldoleal) [inline main.py:131]: Same here.

**Review by beraldoleal — CHANGES_REQUESTED** [review]

**Comment by hdiogenes** [comment]
Superseded by #21.

**Pull request closed without merging**